src/test/ui/suggestions/suggest-move-types.stderr

```diff
@@ -125,6 +125,7 @@ LL | struct Cl<'a, 'b, 'c, T, U, V, M: ThreeWithLifetime<T, 'a, A=(), B=(), C=()
    |                                                        ^^
    |
    = note: type arguments must be provided before lifetime arguments
+   = help: Wrong parameter order found. A possible order is: 'a, 'b, 'c, Self, T, U, V
```

Comment:
How about something like the following:
```suggestion
   = help: reorder the arguments: lifetimes, then types: `<'a, 'b, 'c, T, U, V>`
```
as in:
https://github.com/rust-lang/rust/blob/a59264b01247836c70e24217e0d346b868387525/src/librustc_ast_passes/ast_validation.rs#L745-L753
You'll need to ignore the `Self` type argument.